ivy/array/random.py

```diff
@@ -372,5 +372,25 @@ def shuffle(
         -------
         ret
             An array object, shuffled along the first dimension.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([5,2,9]),
```

Comment:
The examples here are incorrect unfortunately. 😅 
Please refer to the [Docstring Examples](https://lets-unify.ai/ivy/deep_dive/docstring_examples.html#docstring-examples) section of the docs, or see point `6.j` of the checklist for correct instruction on what the example for the `array` class should be like.

Reply:
Done, reformatted these examples to Array Instance Method